pallets/gear/src/benchmarking/code.rs

```diff
@@ -71,6 +71,9 @@ pub struct ModuleDefinition {
     /// Function body of the exported `handle` function. Body is empty if `None`.
     /// Its index is `imported_functions.len() + 1`.
     pub handle_body: Option<FuncBody>,
+    /// Function body of the exported `handle_reply` function. Body is missing if `None`.
+    /// Its index is `imported_functions.len() + 2`.
+    pub reply_body: Option<FuncBody>,
     /// Function body of a non-exported function with index `imported_functions.len() + 2`.
```

Comment:
```suggestion
    /// Function body of a non-exported function with index `imported_functions.len() + 3`.
```

Reply:
The index of the `handle` is `imported_functions.len() + 1`, so this I set to `handle`'s index + 1 = `imported_functions.len() + 2`. Am I wrong?

Reply:
I think better to make empty `handle_reply` if body is None.

Reply:
ok